perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -377,10 +377,13 @@ sub populate_dd_details : Private {
     $c->stash->{reference} = $c->cobrand->call_hook( 'waste_dd_payment_ref' => $p ) || 'GGW' . $c->stash->{property}{uprn};
     $c->stash->{lookup} = $reference;
     $c->stash->{payment_date} = $dt;
+    $c->stash->{start_date} = $dt->ymd;
     $c->stash->{day} = $dt->day;
     $c->stash->{month} = $dt->month;
     $c->stash->{month_name} = $dt->month_name;
     $c->stash->{year} = $dt->year;
+
+    $c->stash->{redirect} = $c->cobrand->call_hook( 'garden_waste_dd_redirect_url' => $p ) || '';
```

Comment:
Nothing appears to happen with this so I assume unfinished? Should we take this out of this commit for now, what's the default redirect, does it work without this parameter?

Reply:
oh, I forgot to commit the redirect param in the form field. I will push a fixup.

By default there is no redirect, you just get a bottomline confirmation page with no link back to Waste so I think we do need it.